lib/Caxy/HtmlDiff/HtmlDiffConfig.php

```diff
@@ -74,6 +74,11 @@ class HtmlDiffConfig
     protected $cacheProvider;
 
     /**
+     * @var null|String
```

Comment:
`String` should be lowercase - it actually assumes it's a class called `String` otherwise